src/CommunityToolkit/Xamarin.CommunityToolkit/Behaviors/Validators/ValidationBehavior.shared.cs

```diff
@@ -15,6 +16,8 @@ public abstract class ValidationBehavior : BaseBehavior<VisualElement>
 
 		public const string InvalidVisualState = "Invalid";
 
+		public ValidationBehavior() => DefaultForceValidateCommand = new AsyncValueCommand(ForceValidate);
```

Comment:
Why not ```protected virtual ICommand DefaultForceValidateCommand => new AsyncValueCommand(ForceValidate);```

Reply:
DefaultForceValidateCommand is called only once during bindable property initialization anyway

Reply:
This is for performance.

Expression Bodied Properties, e.g. `=>`, are like methods; they run every time they are called. 

Before this change, every time `DefaultForceValidateCommand` is called, it will initialize a new `AsyncValueCommand`. 

After this change, `DefaultForceValidateCommand` is initialized once and each time it is called afterwards, it will always return the same reference.

Reply:
As I mentioned, this method is called only **once** by bindable property initializer.

Reply:
It is used as defaultValueCreator for ForceValidateCommandProperty.

Users interacts with **ForceValidateCommand** not **Default**ForceValidateCommand